pokemon/convert.py

```diff
@@ -66,23 +57,21 @@ def convert_image_to_ascii(image, new_width):
     len_pixels_to_chars = len(pixels_to_chars)
 
     image_ascii = [pixels_to_chars[index: index + new_width*2] for index in
-            xrange(0, len_pixels_to_chars, new_width*2)]
-
+            range(0, len_pixels_to_chars, new_width*2)]
```

Comment:
Good call! I was just listening to a talk from one of the python maintainers about how xrange was changed to range, and the choice of xrange for a function name was a terrible idea in the first place!